### ISSUE TITLE
toggle time bubble ON/OFF while at full HPs

### ISSUE THREAD

**Issue by igroglaz** [draconisPW/PWMAngband#313]
atm we got
```
[Escape]=h0[Escape] - disable time bubbles
[Escape]=hn[Escape] - set threshold to (10 x n)% hp (1 <= n <= 9)
[Escape]=h=[Escape] - set 100% threshold
[Escape]=h-[Escape] - toggle between 0 and your setting
```
it's awesome, but it will be super cool to have an option to turn ON/OFF time bubble independently of player's HPs. So even if player got 100% HPs he could toggle time bubble if he is at dangerous level and want to have more time to think. so it could be something like:

`[Escape]=h+[Escape] - toggle ON/OFF time bubble`

**Comment by igroglaz** [comment]
is will be especially useful at endgame - right after you entered stairs and appeared on new lvl

**Comment by draconisPW** [comment]
Err.. you just do =h= to toggle ON then =h- to toggle OFF then =h- to toggle ON again.

**Comment by igroglaz** [comment]
Yep, but as there is this option available:
`[Escape]=h-[Escape] - toggle between 0 and your setting`
it will be great to have the same option for toggling ON/OFF with one hotkey.... atm to toggle ON/OFF you need to use two different hotkeys. While `[Escape]=h-[Escape] - toggle between 0 and your setting` gives possibility to use ony hotkey.

**Comment by draconisPW** [comment]
No you don't. You do "=h=" once and for all to toggle ON then "=h-" to toggle OFF. Then you macro "=h-" to a hotkey and you only press that to toggle ON/OFF.

**Comment by igroglaz** [comment]
it works, thanks :D now we could slow down time when it's necessary! Yay!

But it brings another problem - with this toggling feature player will use two values: 0 or 10.. so if player will be suddenly attacked  by hounds or nasty magic and his HP will go to 30% - player has to react on it manually and turn on time bubble.

Is it possible to add an option (maybe to 'Mangband option' section) with hardcoded auto-time-bubble at 50%? So player will be able to control the buble with toggle hotkey when everything is more or less calm; but if player will be suddenly attacked - this option will slow down game automatically. It's like safety fuse

**Comment by igroglaz** [comment]
But it ​brings another problem - with this toggling feature player will use two values: 0 or 10.. so if player will be suddenly attacked  by hounds or nasty magic and his HP will go to 50% - player has to react on it manually and turn on time bubble.

Please is it possible to add one more value like this:
[Escape]=h-[Escape] - toggle between 0 and your setting
but make it this:
[Escape]=h+[Escape] - toggle between **10** and your setting

So eg my default value is 5. Then most times I have 5 as regular value.. But if I need slow down time when I'm at full HPs - I could 'toggle' to 10 with one hotkey.

This one:
[Escape]=h-[Escape] - toggle between 0 and your setting
is useless.. as toggle between 0 and 5 doesn't really make sense as no one wants to have 0 value at time bubble warning ;)

**Comment by igroglaz** [comment]
still actual problem

**Comment by igroglaz** [comment]
turn ON/OFF for time bubble works and I'll start using it finally. But the feature which I wrote about in this topic above doesn't work properly without this feature:

[Escape]=h+[Escape] - toggle between **10** and your setting

because:

1) I made as you said: "_You do "=h=" once and for all to toggle ON then "=h-" to toggle OFF. Then you macro "=h-" to a hotkey and you only press that to toggle ON/OFF."_

So now I can switch between 6 (my value) and 0 - to turn off time-bubble when I don't need it (eg when character poisoned on low hps and no monsters around).

2) now considering situation when player want to turn off time bubble on demand - even with full hp (eg in case you entered dangerous vault) - when following your istructions I use _"[Escape]=h=[Escape] - set 100% threshold"_ ... But then it will disable previous function - toggling between 6 and 0 :(
